core/bin/external_node/src/config/mod.rs

```diff
@@ -1523,17 +1523,7 @@ impl From<&ExternalNodeConfig> for TxSenderConfig {
             chain_id: config.required.l2_chain_id,
             // Does not matter for EN.
             whitelisted_tokens_for_aa: Default::default(),
-            timestamp_asserter_params: config.remote.l2_timestamp_asserter_addr.map(|address| {
-                TimestampAsserterParams {
-                    address,
-                    min_time_till_end: Duration::from_secs(
-                        config
-                            .optional
-                            .timestamp_asserter_min_time_till_end_sec
-                            .into(),
-                    ),
-                }
-            }),
+            timestamp_asserter_params: None,
```

Comment:
why timestamp_asserter_params is not set?

Reply:
Is it not proper to set at this point? I see lots of other address being set

Reply:
Theoretically yes. but we can set it here.